pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -181,7 +181,9 @@ func (c *Cluster) checkHealth() error {
 			logger.Warningf("mon %q not found in quorum, waiting for timeout (%d seconds left) before failover", mon.Name, timeToFailover)
 
 			// Restart the mon if it is stuck on a failed node
-			c.restartMonIfStuckTerminating(mon.Name)
+			if err := c.restartMonIfStuckTerminating(mon.Name); err != nil {
+				return err
```

Comment:
In this case let's actually log the error instead of returning it. This is a non-critical function that shouldn't cause an error in the overall function.

Reply:
Done